src/main/java/com/careerfair/q/controller/student/implementation/StudentControllerImpl.java

```diff
@@ -48,7 +46,7 @@ public AddStudentResponse addStudent(@RequestBody AddStudentRequest addStudentRe
         return studentService.addStudent(addStudentRequest);
     }
 
-    @PutMapping("/upload-resume/{id}")
+    @PutMapping("/upload-resume/{id}/{email}")
```

Comment:
The URL should be /id/{id}/email/{email} so that the user of this URL knows in which order the things to pass in

Reply:
i didnt work on this but i think it got messed up while moving things around